files/default/templates/hopsworks.slack.tmpl

```diff
@@ -24,10 +24,16 @@
       - *Feature store:* {{ .Labels.featureStore }}
       - *Feature group:* {{ .Labels.featureGroup }}
       - *Feature group version:* {{ .Labels.featureGroupVersion }}
+     {{- else if .Labels.featureMonitorConfig }}
```

Comment:
Don't we show Feature store and Feature group for  featureMonitor alert why is this else if?